assets/Essentials/Voice Models.md

```diff
@@ -95,6 +96,51 @@ If you're curious about the epochs, <u>[learn more here](https://docs.aihub.wtf/
 
 - If it only exists in weights.gg, download the .ZIP & <u>[upload it to HF](https://docs.aihub.wtf/essentials/voice-models/#uploading-to-hugging-face)</u>.  
 
++++ applio.org   
+:::content-center  
+<img src="../searchrvcmodels-img/applio-web-2.png" alt="image" width="350" height="auto">           
+:::  
+
+- This a website where people can find voice models.
+- Not only will you be able to **get models**, but **also guides and use the Applio HF space**, as well as **get access to the Applio API** with **more than 21k models for free**.

```

Comment:
The English is kinda broken, should have proper grammar and make sense before going online

Reply:
Yes, as you understand, it is not my native language.